### DISCUSSION TITLE
캐릭터가 표시가 안되고, state.players.count가 0이 나오는 데 왜 그러는 걸까요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#460] [5. Q&A]
캐릭터가 표시가 안됩니다.

zepeto-multiplay-example의 SampleScene을 그대로 카피해서 따라 만들어보고 있는데, OnStateChange의 state.players.Count가 계속 0으로 되어버립니다.

해당 zepeto-multiplay-example 프로젝트에서는 
- ClientStarter
- WorldMultiplay
- ZepetoPlayers
이 3개만 복사해서 새로운 씬에 넣어도 캐릭터가 표시가 되고 state.players.Count가 1이 되는데

새로운 프로젝트를 만들고
동일한 스크립트를 만들어서 넣어도 state.players.Count가 0이 됩니다.

혹시 프로젝트에서 다른 설정이 필요한 걸까요?

**Comment by minjujuu** [comment]
World.Multiplay 폴더에 있는 index.ts랑 schemas도 같이 잘 옮겨졌을까요?
![image](https://user-images.githubusercontent.com/79794414/156993637-fcbeba0f-a86e-4c33-8ada-90b644003db4.png)

**Reply by ryusiin** [reply]
감사합니다! 

index.ts 문제였어요!
schemas만 적용하고, index.ts를 적용 안했습니다.

동일하게 적용하니, 문제 없이 실행되었습니다:>!
